admin/pre_reg/views.py

```diff
@@ -102,10 +102,9 @@ class DraftDetailView(PreregAdmin, DetailView):
     context_object_name = 'draft'
 
     def get_object(self, queryset=None):
+        draft = DraftRegistration.load(self.kwargs.get('draft_pk'))
```

Comment:
Default for `.get` and `.load`
![screen shot 2016-06-21 at 1 31 01 pm](https://cloud.githubusercontent.com/assets/5169088/16239775/73efb090-37b4-11e6-8f28-cff21091b125.png)
 is `None`